wordpress.yaml

```diff
@@ -1,7 +1,85 @@
 ---
 - name: Deploy Wordpress
   hosts: wordpress
+  become: yes
   tasks:
+    - name: Update system
+      apt:
+        update_cache: yes
 
-    - name: message to remove
-      command: echo 'Here you gonna put all tasks and roles that gonna setup Wordpress application'
+    - name: Upgrade system
+      apt:
+        upgrade: dist  
+
+    - name: Install software 
+      apt:
+        name: 
+          - php-mysql 
+          - php-curl 
+          - php-gd 
+          - php-intl 
+          - php-mbstring 
+          - php-soap 
+          - php-xml 
+          - php-xmlrpc 
+          - php-zip 
+          - nginx 
+          - php7.4-fpm
+
+    - name: Install wp-cli 
+      get_url:
+        url: https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cli.phar
+        dest: /usr/local/bin/wp
+        mode: '0755'
+    
+    - name: Download WP
+      get_url: 
+        url: https://wordpress.org/wordpress-5.7.1.zip
+        dest: /var/www
+    - name: Extract wp
+      unarchive: 
+        src: /var/www/wordpress-5.7.1.zip
+        dest: /var/www/
+    
+    - name: Copy over wordpress nginx config
+      copy:
+        src:  /vagrant/wordpress
+        dest: /etc/nginx/sites-available
+
+    - name: Link sites-available to enabled 
+      file:
+        state: link
+        src: /etc/nginx/sites-available/wordpress
+        dest: /etc/nginx/sites-enabled/wordpress
+    
+    - name: Remove default symbolic link
+      file:
+        path: /etc/nginx/sites-enabled/default
+        state: absent
+    
+    - name: Copy over wp-config
+      copy:
+        src: /vagrant/wp-config.php
+        dest: /var/www/wordpress
+      notify:
+      - Restart nginx
```

Comment:
why do you restart nginx on changing wordpress files instead of trigger restart on changing nginx configuration?